docs/migrate/migrate-subgraph.md

```diff
@@ -0,0 +1,152 @@
+---
+sidebar_position: 2
+title: Migrate from TheGraph
+---
+
+# Migrate from TheGraph
+
+This guide walks through the steps to migrate a subgraph to Subsquid. In what follows we will convert the [Gravatar](https://github.com/graphprotocol/example-subgraph) subgraph into a squid and run it locally. Impatient readers may clone the migrated squid from the `gravatar-squid` branch of the [squid-evm-template repo](https://github.com/subsquid/squid-evm-template):
+
+```bash
+git clone -b gravatar-squid https://github.com/subsquid/squid-evm-template.git
+```
+and run it by following the instructions in README.
+
+Subsquid offers batched processing and a slightly different log handling programming model, which pays off with up to a 100x increase in the indexing speed. At the same time, the concepts of the [schema file](/develop-a-squid/schema-file), code generation and automatic GraphQL API should be familiar for subgraph developers. In most cases the schema file can used by a squid as is. 
+
+## Squid setup 
+
+### 1. Clone template
+
+To begin with, clone the `ethereum-minimal` template from the [squid-evm-template repo](https://github.com/subsquid/squid-evm-template):
+
+```bash
+git clone -b ethereum-minimal https://github.com/subsquid/squid-evm-template.git
+cd squid-evm-template
+```
+
+### 2. Copy the schema file and generate entities
+
+The minimal template already contains a dummy `schema.graphql` file. We replace it with the subgraph's schema as is:
+
+```gql file=schema.graphl
+type Gravatar @entity {
+  id: ID!
+  owner: Bytes!
+  displayName: String!
+  imageUrl: String!
+}
+```
+
+Next, we generate the entities from the schema and check the squid builds:
+```bash
+make codegen
+make build
+```
+
+After that start the local database and generate the migrations from the generated entities:
+```bash
+make up
+make migration
+```
+The migration file will appear in `db/migrations`
+
+### Generate typings from ABI
+
+Copy the contract ABI (`Gravity.json`) to `src/abi` and run [EVM typegen](/develop-a-squid/typegen/squid-evm-typegen):
+```bash
+npx squid-evm-typegen --abi=src/abi/Gravity.json --output=src/abi/Gravity.ts
+```
+
+The generated access class will be generated in `src/abi/Gravity.ts`. In particular, it will contain the boilerplate to decode the logs and the topic definitions used at the next step. 
```

Comment:
This might be something you are already aware of, but "generated access class will be generated" is a bit of a repetition.
Might suggest:
"The generated access class can be found in", or
"An access class will be generated in"

I prefer the first option